integration_tests/test_commands/policies.yaml

```diff
@@ -0,0 +1,7 @@
+---
+- Version: "2012-10-17"
+  Statement:
+    - Effect: Deny
```

Comment:
This will prevent codebuild from being able to write logs to cloudwatch. Can we add support for no additional policies? that can even be the default.